### DISCUSSION TITLE
Version 4.8.0 alpha2 not on Docker Hub

### DISCUSSION THREAD

**Discussion by oliv10** [wazuh/wazuh#21334] [Q&A] [answered]
This version is not on the docker hub and I would like to try it out. When will it be available?

**Comment by c-bordon** [comment, marked as answer]
Hello Oliver, currently we have found a change in dependencies of Wazuh manager (xz-utils), so we have not been able to create Docker images for this stage (Alpha 2), as Alpha 2 is an early testing phase, the Docker images will be created for Beta 1 stage:

You can follow the Alpha 2 testing progress at this issue: https://github.com/wazuh/wazuh/issues/21386
